drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -186,16 +174,13 @@ function DragAndDropTemplates(configuration) {
     var zoneTemplate = function(zone, ctx) {
         var className = ctx.display_zone_labels ? 'zone-name' : 'zone-name sr';
         var selector = ctx.display_zone_borders ? 'div.zone.zone-with-borders' : 'div.zone';
-
+        
         // If zone is aligned, mark its item alignment
         // and render its placed items as children
         var item_wrapper = 'div.item-wrapper';
-        var items_in_zone = [];
-        if (zone.align !== 'none') {
-            item_wrapper += '.item-align.item-align-' + zone.align;
-            var is_item_in_zone = function(i) { return i.is_placed && (i.zone === zone.uid); };
-            items_in_zone = $.grep(ctx.items, is_item_in_zone);
-        }
+        item_wrapper += '.item-align.item-align-' + zone.align;
```

Comment:
@e-kolpakov A couple nits:

* Maybe define `item_wrapper` in a single line
* Update the comment above to reflect the fact that all zones use alignment now